### ISSUE TITLE
Missing token timestamps/offsets for repetitive stretches

### ISSUE THREAD

**Issue by smoores-dev** [ggml-org/whisper.cpp#2329]
When recognizing some border-line pathological audio content, apparently Whisper.cpp sometimes will output tokens without `timestamps` or `offsets` properties. Attached is a sample file, a TTS narration of a passage that includes, at a few points, the phrase "I love you" several dozen times in a row.

https://github.com/user-attachments/assets/0ffc8cc5-ef80-409e-8ed7-cf9441f6bbc1

[00000-00009.wav.json](https://github.com/user-attachments/files/16435831/00000-00009.wav.json)

Command to produce the flawed output:

```sh
main -m ggml-tiny.bin --output-json-full --language en --threads 4 --processors 1 --no-gpu --max-len 0 --best-of 5 --beam-size 5 --entropy-thold 2.4 ~/Downloads/00000-00009.wav
```

Sifting through the resulting JSON (also attached) reveals the problem token: at the end of the first very long string of "I love you"s, the last "you" token looks like this:

```json
{
	"text": " you",
	"id": 291,
	"p": 0.960787,
	"t_dtw": -1
}
```

It has neither timestamps nor offsets.

This is sort of odd; I'm not sure whether the model output is omitting this information, or if there's some other bug, but it makes it hard to parse whisper.cpp outputs when properties can go missing this way. See also https://github.com/echogarden-project/echogarden/issues/65.

**Comment by smoores-dev** [comment]
@ggerganov do you have any thoughts on whether this should be considered a bug or not?

**Comment by ggerganov** [comment]
I downloaded the json, but I don't see the entry that is missing the timestamps. On which line in the file is it located?

**Comment by smoores-dev** [comment]
Ah yeah sorry about that! It's line 16272
